### DISCUSSION TITLE
Debugger Variables

### DISCUSSION THREAD

**Discussion by warpedghost** [microsoft/vscode-discussions#969] [Extension Development QnA] [answered]
Something weird has happened to my debugger.  All of my variables in the debugger are now 1 letter names instead of the whole variable name.  Does anyone know why this is happening? 
![debugger](https://github.com/microsoft/vscode-discussions/assets/48892128/9b578b54-3147-44ac-8d0d-265b3f141b66)

**Comment by gjsjohnmurray** [comment, marked as answer]
Are you developing an extension? If not, this discussion forum is the wrong place to be asking. Please see the READ FIRST post that is pinned to the top.
